src/Windows/Containers/Test/RegistryKeyMock.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Common.Core.OS;
+using Microsoft.Win32;
+
+namespace Microsoft.R.Containers.Windows.Test {
+    class RegistryKeyMock : IRegistryKey {
```

Comment:
We have registry mocks in `Microsoft.Common.Core.Test` project. You can simply reference it.